src/app/SIPUserAgents/SIPNotifierClient.cs

```diff
@@ -96,6 +96,20 @@ public string CallID
             m_resourceURI = resourceURI.CopyOf();
             m_subscribeCallID = CallProperties.CreateNewCallId();
             m_subscriptionFromTag = CallProperties.CreateNewTag();
+
+            m_sipTransport.SIPTransportRequestReceived += async (lep, rep, req) =>
+            {
+                if (req.Method == SIPMethodsEnum.NOTIFY)
+                {
+                    await GotNotificationRequest(lep, rep, req);
+                }
+                else
+                {
+                    SIPResponse notSupported = SIPResponse.GetResponse(req, SIPResponseStatusCodesEnum.MethodNotAllowed, null);
+                    await sipTransport.SendResponseAsync(notSupported);
+                }
+            };
+

```

Comment:
This block isn't necessary. https://github.com/sipsorcery-org/sipsorcery/blob/b9103ea2a2a3321f5cd315dd939c7ef03c7b6470/src/app/SIPUserAgents/SIPNotifierClient.cs#L105 was added this morning to do the same thing.

Reply:
In your change, GotNotificationRequest is called for all Methods, but it should only be called for Notification.

Reply:
It's no different to the lambda method. In both cases the `SIPTransportRequestReceived` event has to be caught and the SIP request method checked. It's the first check done in `GetNotificationResult`.

https://github.com/sipsorcery-org/sipsorcery/blob/b9103ea2a2a3321f5cd315dd939c7ef03c7b6470/src/app/SIPUserAgents/SIPNotifierClient.cs#L143

Reply:
So, my PR can be deleted as things are done